pkgs/top-level/all-packages.nix

```diff
@@ -24550,6 +24550,8 @@ with pkgs;
 
   asterisk-module-sccp = callPackage ../servers/asterisk/sccp { };
 
+  asterisk-ldap = lowPrio (asterisk.override { ldapSupport = true; });
```

Comment:
why the `lowPrio`?

Reply:
because if there is no prio set and there is more then one bin with the same name from different packages. It will link the one with the higher prio, if you ripgrep through nixpkgs it seems like setting lowPrio for overrides is commen practice.